x-pack/plugins/index_management/public/application/components/mappings_editor/components/load_mappings/load_mappings_provider.tsx

```diff
@@ -224,7 +224,6 @@ export const LoadMappingsProvider = ({ onJson, esNodesPlugins, children }: Props
           maxWidth={600}
         >
           {view === 'json' ? (
-            // The CSS override for the EuiCodeEditor requires a parent .application css class
             <div className="application">
```

Comment:
If I understand the deleted comment correctly, shouldn't we also remove the classname?